tests/tests/ray_tracing_use_after_free/mod.rs

```diff
@@ -0,0 +1,128 @@
+use std::{iter, mem};
+use wgpu::ray_tracing::{
+    AccelerationStructureUpdateMode, BlasBuildEntry, BlasGeometries, BlasTriangleGeometry,
+    CommandEncoderRayTracing, DeviceRayTracing, TlasInstance, TlasPackage,
+};
+use wgpu::util::{BufferInitDescriptor, DeviceExt};
+use wgpu::{
+    include_wgsl, BindGroupDescriptor, BindGroupEntry, BindingResource, ComputePassDescriptor,
+    ComputePipelineDescriptor,
+};
+use wgpu_macros::gpu_test;
+use wgpu_test::{GpuTestConfiguration, TestParameters, TestingContext};
+use wgt::{
+    AccelerationStructureFlags, AccelerationStructureGeometryFlags, BlasGeometrySizeDescriptors,
+    BlasTriangleGeometrySizeDescriptor, BufferAddress, BufferUsages, CommandEncoderDescriptor,
+    CreateBlasDescriptor, CreateTlasDescriptor, VertexFormat,
+};
+
+fn required_features() -> wgpu::Features {
+    wgpu::Features::RAY_QUERY | wgpu::Features::RAY_TRACING_ACCELERATION_STRUCTURE
+}
+
+fn execute(ctx: TestingContext) {
```

Comment:
It would be great to get a bit of commentary within the function what exactly we're testing here and which points are important to look at. I personally like breaking up the test into sections delinted by comments, this helps make clear what is going on.

Reply:
I think I've added enough commentary for this.